frontend/javascripts/oxalis/model/volumetracing/volumelayer.js

```diff
@@ -324,32 +345,36 @@ class VolumeLayer {
     ) {
       return null;
     }
-
-    const normedPerpendicularVector = this.vector2PerpendicularVector(
+    let [xa, ya, xb, yb, xc, yc, xd, yd] = this.rectangleToCircles(
       floatingCoord2dLastPosition,
       floatingCoord2dPosition,
-    );
-    const shiftVector = this.vector2WithScale(
-      this.vector2ScalarMultiplication(normedPerpendicularVector, radius),
+      radius,
       scale,
     );
-    const negShiftVector = this.vector2ScalarMultiplication(shiftVector, -1);
-
-    // calculate the rectangle's corners
-    const [ax, ay] = this.vector2Sum(floatingCoord2dPosition, negShiftVector);
-    const [bx, by] = this.vector2Sum(floatingCoord2dPosition, shiftVector);
-    const [cx, cy] = this.vector2Sum(floatingCoord2dLastPosition, shiftVector);
-    const [dx, dy] = this.vector2Sum(floatingCoord2dLastPosition, negShiftVector);
-
-    const minCoord2d = [Math.floor(Math.min(ax, bx, cx, dx)), Math.floor(Math.min(ay, by, cy, dy))];
-    const maxCoord2d = [Math.ceil(Math.max(ax, bx, cx, dx)), Math.ceil(Math.max(ay, by, cy, dy))];
+    const minCoord2d = [Math.floor(Math.min(xa, xb, xc, xd)), Math.floor(Math.min(ya, yb, yc, yd))];
+    const maxCoord2d = [Math.ceil(Math.max(xa, xb, xc, xd)), Math.ceil(Math.max(ya, yb, yc, yd))];
     const [width, height] = maxCoord2d;
     const map = this.createMap(width, height);
 
     const setMap = (x, y) => {
       map[x][y] = true;
     };
-    Drawing.fillRectangle(ax, ay, bx, by, cx, cy, dx, dy, setMap);
+
+    // translate the coordinates so the containing box originates in (0|0)
+    const [diffX, diffY] = [
+      Math.floor(Math.min(xa, xb, xc, xd)),
+      Math.floor(Math.min(ya, yb, yc, yd)),
+    ];
```

Comment:
Isn't this the same as `minCoord2d`?

```suggestion
    const [diffX, diffY] = minCoord2d;
```

Reply:
Oh, yes, it is!